nilearn/decomposition/tests/test_decomposition_estimators.py

```diff
@@ -23,7 +23,6 @@
 )
 
 ESTIMATORS_TO_CHECK = [
-    _MultiPCA(verbose=0),
```

Comment:
not need to test private classes that will anyway be tested when we test the public ones that inherit from it.